Woof/WoofTests/ViewModel/LoginViewModelTests.swift

```diff
@@ -0,0 +1,16 @@
+import XCTest
+
+final class LoginViewModelTests: XCTestCase {
```

Comment:
Add more tests for properties:

- @Published var isTheOwnerRoleSelected
- @Published var isTheSitterRoleSelected

Reply:
Done.